Specs/Scene/GltfLoaderSpec.js

```diff
@@ -1657,6 +1660,287 @@ describe(
+        expect(toroidalNormalized.hasValueTransform).toBe(true);
+        expect(toroidalNormalized.offset).toBe(0);
+        expect(toroidalNormalized.scale).toBe(0.034482758620689655);
+
+        const poloidalNormalized = propertyAttribute.getProperty(
+          "poloidalNormalized"
+        );
+        expect(poloidalNormalized.attribute).toBe("_FEATURE_ID_1");
+        expect(poloidalNormalized.hasValueTransform).toBe(true);
+        expect(poloidalNormalized.offset).toBe(0);
+        expect(poloidalNormalized.scale).toBe(0.05263157894736842);
+
+        // These two properties have offset/scale in both the class definition
+        // and the property attribute. The latter should be used.
+        const toroidalAngle = propertyAttribute.getProperty("toroidalAngle");
+        expect(toroidalAngle.attribute).toBe("_FEATURE_ID_0");
+        expect(toroidalAngle.hasValueTransform).toBe(true);
+        expect(toroidalAngle.offset).toBe(0);
+        expect(toroidalAngle.scale).toBe(0.21666156231653746);
+
+        const poloidalAngle = propertyAttribute.getProperty("poloidalAngle");
+        expect(poloidalAngle.attribute).toBe("_FEATURE_ID_1");
+        expect(poloidalAngle.hasValueTransform).toBe(true);
+        expect(poloidalAngle.offset).toBe(-3.141592653589793);
+        expect(poloidalAngle.scale).toBe(0.3306939635357677);
+      });
+    });
+
+    it("loads BoxTexturedWithPropertyAttributes", function () {
+      return loadGltf(boxWithPropertyAttributes).then(function (gltfLoader) {
+        const components = gltfLoader.components;
+        const scene = components.scene;
+        const nodes = components.nodes;
+        const rootNode = scene.nodes[0];
+        const childNode = rootNode.children[0];
+        const primitive = childNode.primitives[0];
+        const attributes = primitive.attributes;
+        const positionAttribute = getAttribute(
+          attributes,
+          VertexAttributeSemantic.POSITION
+        );
+        const normalAttribute = getAttribute(
+          attributes,
+          VertexAttributeSemantic.NORMAL
+        );
+        const texcoordAttribute = getAttribute(
+          attributes,
+          VertexAttributeSemantic.TEXCOORD,
+          0
+        );
+        const warpMatrixAttribute = getAttributeByName(
+          attributes,
+          "_WARP_MATRIX"
+        );
+        const temperaturesAttribute = getAttributeByName(
+          attributes,
+          "_TEMPERATURES"
+        );
+
+        const indices = primitive.indices;
+        const material = primitive.material;
+        const metallicRoughness = material.metallicRoughness;
+
+        // prettier-ignore
+        const rootMatrix = new Matrix4(
+          1.0, 0.0, 0.0, 0.0,
+          0.0, 0.0, 1.0, 0.0,
+          0.0, -1.0, 0.0, 0.0,
+          0.0, 0.0, 0.0, 1.0
+        );
+
+        const childMatrix = Matrix4.IDENTITY;
+
+        expect(rootNode.children.length).toBe(1);
```

Comment:
You could probably remove the node-related tests here because the node structure gets tested in another test in this file. This test should focus more on the attributes and metadata